packages/twenty-server/src/engine/api/rest/core/rest-api-core-v2.service.ts

```diff
@@ -0,0 +1,63 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+
+import { Request } from 'express';
+import { capitalize } from 'twenty-shared';
+
+import { CoreQueryBuilderFactory } from 'src/engine/api/rest/core/query-builder/core-query-builder.factory';
+import { parseCorePath } from 'src/engine/api/rest/core/query-builder/utils/path-parsers/parse-core-path.utils';
+import { extractObjectIdFromPath } from 'src/engine/api/rest/core/utils/extract-id-from-path.utils';
+import { TwentyORMGlobalManager } from 'src/engine/twenty-orm/twenty-orm-global.manager';
+
+@Injectable()
+export class RestApiCoreServiceV2 {
+  constructor(
+    private readonly coreQueryBuilderFactory: CoreQueryBuilderFactory,
+    private readonly twentyORMGlobalManager: TwentyORMGlobalManager,
+  ) {}
+
+  async delete(request: Request) {
+    const { workspace } = request;
+    const { object: parsedObject } = parseCorePath(request);
+    const objectId = extractObjectIdFromPath(request.path);
```

Comment:
think you can simplify here
```suggestion
    const { object: parsedObject, id: recordId } = parseCorePath(request);
```

Reply:
Looks clearer, thanks